sites/dotnet-guide/contents/tutorials/basics/index.md

```diff
@@ -0,0 +1,17 @@
+---
+date: 2020-06-05
+title: The Basics Series
+technologies: [asp.net, .net, csharp]
+topics: [web, data]
+author: khalidabuhakmeh
+subtitle: A set of short basic walkthroughs of different .NET technologies
+thumbnail: ./thumbnail.png
+tutorialItems:
+  - /tutorials/basics/aspnet-mvc/
+  - /tutorials/basics/razor-pages/
+  - /tutorials/basics/entity-framework-core/
+  - /tutorials/basics/ado-net/
+  - /tutorials/basics/dapper/
+---
+
+In this series of videos, we explore the absolute basics of technologies in .NET. From web technologies to data access mechanisms. These videos display the bare minimum anyone needs to get started with tech, and videos range from 1 to 3 minutes in length.
```

Comment:
Perhaps the wording on this could be cleaned up a bit:
- The second sentence is more like a fragment, perhaps join to the first
- "get started with tech" sounds somewhat awkward